docs/content/mobile/android/local-deploy.md

```diff
@@ -1,73 +1,65 @@
-  1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
-  1. Open the project that you want to deploy with the Unity Editor and go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window.
-  1. Open the scene that starts both your [client-]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker).
-  1. In the **Unity Remote** section, click on the drop-down menu beside the **Device** option and select **Any Android Device**.
-  1. On your mobile device, open the **Unity Remote** app. Make sure you allow it permissions for location and camera.
-  1. In the Unity Editor, Select **SpatialOS** > **Local launch**.
+## Prepare your project to connect to a local deployment
 
-    > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
+To connect your Android device to a local deployment, you need to prepare; follow the steps below to ensure any local deployment you launch via the Unity Editor is correctly set up.
 
-  1. In the Editor’s Game view, select **Play**.
+1. Open your project in the Unity Editor.
+1. Navigate to **SpatialOS** > **GDK Tools configuration** to open the configuration window.
+1. In the **Runtime IP for local deployment** field, enter your local machine's IP address. (You can find how to do this on the [Lifehacker website](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address).)
+1. Select **Save** and close the window.
 
-    > You can change the resolution of the Game view in your Unity Editor to make sure it does not appear stretched on your mobile device. Choose the resolution that’s identical to your mobile device to produce the best results.
+## Use Unity Remote
 
-  1. You should now see your Unity Editor game view mirrored on your Android device.
+You need the Unity Remote app for this. See the [Unity documentation](https://docs.unity3d.com/Manual/UnityRemote5.html) for details.
 
-## Connecting your Android emulator to a local deployment
+1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
+1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
+1. In the Unity Editor, open the project that you want to deploy and go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window.
+1. Open the Scene that starts both your [client-]({{urlRoot}}/content/glossary#client-worker) and [server-workers]({{urlRoot}}/content/glossary#server-worker).
+1. In **Editor Settings** window's **Unity Remote** section, set the **Device** setting to **Any Android Device**.
```

Comment:
Just noticed that you completely rewrote this sentence.
- a `the` is missing in `In **Editor Settings** ..`
- where is this Editor settings window?